### ISSUE TITLE
Add reference to Docker-Mac-Net-Connect to LoadBalancer page

### ISSUE THREAD

**Issue by maucaro** [kubernetes-sigs/kind#3448]
**What would you like to be documented**:
The LoadBalancer [page](https://kind.sigs.k8s.io/docs/user/loadbalancer/) correctly states that "On macOS and Windows, docker does not expose the docker network to the host." It also suggests using Ingress as a workaround. On MacOS, another workaround is to use [Docker-Mac-Net-Connect](https://github.com/chipmk/docker-mac-net-connect) to access the Docker network.

**Why is this needed**:
While Ingress is a suitable workaround for some cases, others may require direct access to a LB. Including this workaround can save time in that users would not need to discover it on their own.

**Comment by professorabhay** [comment]
Hey, @maucaro! I want to work on this.
Could you please share anything else that can help?

**Comment by maucaro** [comment]
> Hey, @maucaro! I want to work on this. Could you please share anything else that can help?

Of course. Here is a [Gist](https://gist.github.com/maucaro/935e4dfa1738e1810d86a82b7cc1a45f) of how I got this to work. Take a look and let me know if you have any questions.

**Comment by BenTheElder** [comment]
I think this is obsoleted by https://github.com/kubernetes-sigs/cloud-provider-kind which we currently document versus the previous metallb docs.
